casanovo/config.py

```diff
@@ -126,3 +126,7 @@ def validate_param(self, param: str, param_type: Callable):
             raise TypeError(
                 f"Incorrect type for configuration value {param}: {err}"
             )
+
+    def items(self) -> tuple[str, ...]:
```

Comment:
Locally failing the unit tests, getting a `TypeError: 'type' object is not subscriptable` because of this line. Passes all tests when `-> tuple[str, ...]` commented out.

Reply:
Ah yeah, I forgot using lowercase `tuple ` is a newer Python feature. Maybe added in 3.10?